compiler/sem/semtypinst.nim

```diff
@@ -121,8 +121,7 @@ proc cacheTypeInst(c: PContext; inst: PType) =
   addToGenericCache(c, gt.sym, inst)
 
 type
-  # TODO: This type should be marked acyclic
-  LayeredIdTable* = ref object
+  LayeredIdTable* {.acyclic.} = ref object
```

Comment:
Same as above

Reply:
Isn't it fair to say here that it's for `nextLayer` and that's a pretty obvious linked list (non-circular)?